docs/source/release_notes/index.rst

```diff
@@ -37,3 +44,16 @@ The following pages summarize what is new in Avocado:
    0_27_0
    0_26_0
    0_25_0
+
+Long Term Stability Releases
+============================
+
+The LTS releases are regular releases considering the release cycle, but
+they are flagged as supported for a period of about 18 months after
```

Comment:
it's about 18 months actually. Since a new release is released every year **or so**. The important/hard time-frame we have is *6 months of support overlap between two subsequent LTSs*.